packages/blocks/src/embed-linked-doc-block/embed-linked-doc-model.ts

```diff
@@ -12,6 +12,7 @@ export const EmbedLinkedDocStyles: EmbedCardStyle[] = [
 
 export type EmbedLinkedDocBlockProps = {
   pageId: string;
+  type: 'Subpage' | 'LinkedPage';
```

Comment:
We should try to remove Subpage in newly created data and code. Is it possible to use only LinkedPage here?

Reply:
I have "Subpage" is for forward compatibility. default prop is "LinkedPage" only
![image](https://github.com/toeverything/blocksuite/assets/54364088/abc9377c-6707-4520-912b-9458f9951777)

Reply:
@doodlewind
done